elements/pfe-navigation/demo/pfe-navigation.story.js

```diff
@@ -92,7 +92,7 @@ stories.add(PfeNavigation.tag, () => {
     })
   }) : "";
 
-  let search = slotCheck.search ? createItem("search", "search", "Search", `<div class="pfe-navigation-item__tray--container">
+  let search = slotCheck.search ? createItem("search", "web-search", "Search", `<div class="pfe-navigation-item__tray--container">
```

Comment:
I don't understand what this is doing, in master the icon that is loaded is:
`https://access.redhat.com/webassets/avalon/j/lib/rh-iconfont-svgs/web-icon-search.svg` 
which is correct

Reply:
All of our icons are prefixed with something and then have a dash. The prefix is the set that the icon is coming from. We have a set called "web" that has the Red Hat web icons. Using `search` doesn't tell `pfe-icon` which set the icon is coming from.

Reply:
@starryeyez024 These are the built-in icons sets that we have.

https://github.com/patternfly/patternfly-elements/blob/master/elements/pfe-icon/src/builtin-icon-sets.js#L16

Reply:
Sorry, I dont understand how it is currently broken in master, it seems to be working already

Reply:
The icons in the story are not working in master. This fixes it.

Reply:
Maybe taking a quick look at the issue might help.
https://github.com/patternfly/patternfly-elements/issues/700